.github/workflows/dependency-review.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@8ca2b8b2ece13480cda6dacd3511b49857a23c09 # v2.5.1
         with:
-          egress-policy: block
+          egress-policy: audit
```

Comment:
Not sure why, but the allow list was reset, so had to move this back to audit. (It was blocking connections to github)